content/tutorials/esc-external-secret-operator/index.md

```diff
@@ -0,0 +1,491 @@
+			Version:   pulumi.String("0.10.4"),
+			Namespace: externalSecretsNamespace.Metadata.Name(),
+			RepositoryOpts: &helm.RepositoryOptsArgs{
+				Repo: pulumi.String("https://charts.external-secrets.io"),
+			},
+		})
+		if err != nil {
+			return err
+		}
+
+		patSecret, err := v1.NewSecret(ctx, "patSecret", &v1.SecretArgs{
+			Metadata: &metav1.ObjectMetaArgs{
+				Namespace: externalSecretsNamespace.Metadata.Name(),
+				Name:      pulumi.String("pulumi-access-token"),
+			},
+			StringData: pulumi.StringMap{
+				"PULUMI_ACCESS_TOKEN": conf.RequireSecret("pulumi-pat"),
+			},
+			Type: pulumi.String("Opaque"),
+		})
+		if err != nil {
+			return err
+		}
+
+		clusterSecretStore, err := apiextensions.NewCustomResource(ctx, "cluster-secret-store", &apiextensions.CustomResourceArgs{
+			ApiVersion: pulumi.String("external-secrets.io/v1beta1"),
+			Kind:       pulumi.String("ClusterSecretStore"),
+			Metadata: &metav1.ObjectMetaArgs{
+				Name: pulumi.String("secret-store"),
+			},
+			OtherFields: kubernetes.UntypedArgs{
+				"spec": pulumi.Map{
+					"provider": pulumi.Map{
+						"pulumi": pulumi.Map{
+							"organization": pulumi.String(ctx.Organization()),
+							"project":      pulumi.String("dirien"),
+							"environment":  pulumi.String("hello-world"),
+							"accessToken": pulumi.Map{
+								"secretRef": pulumi.Map{
+									"name":      patSecret.Metadata.Name(),
+									"key":       pulumi.String("PULUMI_ACCESS_TOKEN"),
+									"namespace": patSecret.Metadata.Namespace(),
+								},
+							},
+						},
+					},
+				}},
+		}, pulumi.DependsOn([]pulumi.Resource{externalSecretsRelease}))
+		if err != nil {
+			return err
+		}
+
+		_, err = apiextensions.NewCustomResource(ctx, "external-secret", &apiextensions.CustomResourceArgs{
+			ApiVersion: pulumi.String("external-secrets.io/v1beta1"),
+			Kind:       pulumi.String("ExternalSecret"),
+			Metadata: &metav1.ObjectMetaArgs{
+				Name:      pulumi.String("secret"),
+				Namespace: pulumi.String("default"),
+			},
+			OtherFields: kubernetes.UntypedArgs{
+				"spec": pulumi.Map{
+					"data": pulumi.Array{
+						pulumi.Map{
+							"secretKey": pulumi.String("esc-secret"),
+							"remoteRef": pulumi.Map{
+								"key": pulumi.String("hello"),
+							},
+						},
+					},
+					"refreshInterval": pulumi.String("20s"),
+					"secretStoreRef": pulumi.Map{
+						"kind": clusterSecretStore.Kind,
+						"name": clusterSecretStore.Metadata.Name(),
+					},
+				},
+			},
+		}, pulumi.DependsOn([]pulumi.Resource{externalSecretsRelease}))
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+}
+```
+
+{{% /choosable %}}
+
+You can then deploy the Pulumi program using below command as you would normally do with any Pulumi program:
+
+```bash
+pulumi up
+```
+
+#### Cleanup
+
+To remove the External Secrets Operator and the created resources, you can use the following commands:
+
+```bash
+pulumi destroy
+```
+
+## Push Secrets to Pulumi ESC
```

Comment:
In this section can you also walk through showing how this works? Like write a dummy k8s secret value, show how to push it, and then maybe a screenshot of it getting successfully pushed to the ESC environment?